python/lsst/pipe/tasks/processCcd.py

```diff
@@ -63,11 +64,13 @@ class ProcessCcdConfig(pexConfig.Config):
     )
 
     def setDefaults(self):
-        self.isr.doWrite = False
-        self.charImage.doWriteExposure = False
-        self.charImage.detection.doTempLocalBackground = False
-        self.calibrate.detection.doTempLocalBackground = False
-        self.calibrate.deblend.maxFootprintSize = 2000
```

Comment:
go to calibrate.CalibrateConfig and add detection.doTempLocalBackground = False and deblend.maxFootprintSize = 2000 into the setDefaults method.